samples/AppConfiguration/Generated/Models/Key.cs

```diff
@@ -16,12 +16,13 @@ internal Key()
         }
 
         /// <summary> Initializes a new instance of Key. </summary>
-        /// <param name="name"></param>
+        /// <param name="name"> The Name. </param>
         internal Key(string name)
         {
             Name = name;
         }
 
+        /// <summary> The Name. </summary>
```

Comment:
this should be "Gets the [lower case property name]" unless its not readonly then "Gets or sets the [lower case property name]"

Reply:
@m-nash Hi Michael, I have modified it according to your request, please take a look 😃  .